Sources/PayPalWebCheckout/PayPalWebCheckoutFundingSource.swift

```diff
@@ -0,0 +1,12 @@
+/// Enum class to specify the type of funding for an order.
+/// For more information go to: https://developer.paypal.com/docs/checkout/pay-later/us/
+public enum PayPalWebCheckoutFundingSource: String {
+    /// PayPal Credit will launch the web checkout flow and display PayPal Credit funding to eligible customers
+    /// Eligible costumers receive a revolving line of credit that they can use to pay over time.
+    case paypalCredit = "credit"
+    /// PayLater will launch the web checkout flow and display Pay Later offers to eligible customers,
+    /// which include short-term, interest-free payments and other special financing options
+    case paylater = "paylater"
+    /// PayPal will launch the web checkout for a one-time PayPal Checkout flow
```

Comment:
```suggestion

    /// PayPal will launch the web checkout for a one-time PayPal Checkout flow
```

Just some nitpicks for readability